main/EDTypesMod.F90

```diff
@@ -421,12 +420,10 @@ module EDTypesMod
      real(r8) ::  spread                                          ! dynamic canopy allometric term [unitless]
 
      ! site-level variables to keep track of the disturbance rates, both actual and "potential"
-     real(r8) :: disturbance_rates_primary_to_primary(N_DIST_TYPES)      ! actual disturbance rates from primary patches to primary patches [m2/m2/day]
-     real(r8) :: disturbance_rates_primary_to_secondary(N_DIST_TYPES)    ! actual disturbance rates from primary patches to secondary patches [m2/m2/day]
-     real(r8) :: disturbance_rates_secondary_to_secondary(N_DIST_TYPES)  ! actual disturbance rates from secondary patches to secondary patches [m2/m2/day]
-     real(r8) :: potential_disturbance_rates(N_DIST_TYPES)               ! "potential" disturb rates (i.e. prior to the "which is most" logic) [m2/m2/day]
-     real(r8) :: primary_land_patchfusion_error                          ! error term in total area of primary patches associated with patch fusion [m2/m2/day]
-     
+     real(r8) :: disturbance_rates(N_DIST_TYPES,n_landuse_cats, n_landuse_cats)  ! actual disturbance rates for each disturbance type  [m2/m2/day]
+     real(r8) :: primary_land_patchfusion_error             ! error term in total area of primary patches associated with patch fusion [m2/m2/day]
+     real(r8) :: landuse_transition_matrix(n_landuse_cats, n_landuse_cats) ! land use transition matrix as read in from HLM and aggregated to FATES land use types [m2/m2/year]
```

Comment:
@ckoven we currently don't use this site-level variable.  Currently, we are defining the same thing locally in the `disturance_rates` subroutine, which is fine since it is only ever needed in that space:
https://github.com/NGEET/fates/blob/150271047867f54ac83e1569db41fb78b33e1986/biogeochem/EDPatchDynamicsMod.F90#L215
That said, is there a potential future in which we'd want to record the matrix for each site?

Reply:
In subsequent developments, I am now using this site-level `landuse_transition_matrix` variable for history output, so let's leave it in this PR even though it is not yet being used here.